doc/nrf/bootloaders: clarify MCUboot as second-stage boot setup (#15742)
nrfconnect/sdk-nrf: doc/nrf/config_and_build/bootloaders/index.rst

     - Dual-slot direct-xip
   * - :doc:`MCUboot <mcuboot:index-ncs>`
     - Yes

>      - Yes, when :ref:`bootloader` is the first-stage

Reviewer: ```suggestion
     - Yes (only with :ref:`NSIB <bootloader>` as first-stage)
```
